docs/app/components/api-property/api-property.component.html

```diff
@@ -0,0 +1,12 @@
+<td class="api-property col-md-3">
+    <span class="highlight">{{ name }}</span>
+</td>
+<td class="api-property col-md-9">
+    <span *ngIf="required" class="italic">Required<br /></span> 
```

Comment:
A couple of things about this:
1. Classes named `italic` and `highlight` are not acceptable. I suggest something like `api-property-metadata-name` and `api-property-metadata-value`.
2. span + br is not the right way to vertically lay these out. Please contain each line within a div that has an appropriate class name. The content should also have a parent div, in case it need particular styling in the future.